src/cli/commands/install.js

```diff
@@ -431,6 +431,10 @@ export class Install {
       this.scripts.setArtifacts(artifacts);
     }
 
+    if (Object.getPrototypeOf(this) === Install.prototype && depRequests.length === 0) {
```

Comment:
Look at code on line 367, it saves explicitly when there is nothing to install.
Maybe that logic should be there?